src/components/CompletedWorkoutBlock.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from 'react';
+import { Text, TouchableOpacity, View } from 'react-native';
+import Ionicon from '@expo/vector-icons/Ionicons';
+import { ConsumableWorkout } from '../types';
+import { CompletedBlockContainer } from './CompletedBlockContainer';
+
+interface CompletedWorkoutBlockProps {
+  workout: ConsumableWorkout;
+}
+
+const CompletedWorkoutBlock: React.FC<CompletedWorkoutBlockProps> = ({ workout }) => {
+  const [name, setName] = useState<string>(workout.name || '');
+
+  // count the # of completed/total sets in a workout
+  // iterates through a workout's exercises, keeping a running count
+
+  function countSets() {
```

Comment:
Could you move this function into `/utils`, maybe make a file called `parsing.ts` and then just have this function take in a `workout: ConsumableWorkout` parameter. I think we will need to use this same logic in a few other places going forward so it'd be nice to have it exported from somewhere!